sfdx-project.json

```diff
@@ -12,7 +12,7 @@
     "sfdcLoginUrl": "https://login.salesforce.com",
     "sourceApiVersion": "54.0",
     "packageAliases": {
-        "PLP Demo (2GP)": "0Ho8c000000XZG5CAO",
-        "PLP Demo (2GP)@1.0.0-1": "04t8c000000ptSUAAY"
+        "PLP Demo": "0Ho8c000000wk2nCAA",
```

Comment:
please remove org id's for the merge to main, and then could restore for local testing

Reply:
Done